WooCommerce/Resources/AppStoreStrings.pot

```diff
@@ -57,13 +57,11 @@ msgctxt "app_store_promo_text"
 msgid "Run your store from anywhere"
 msgstr ""
 
-msgctxt "v8.3-whats-new"
+msgctxt "v8.4-whats-new"
 msgid ""
-"Merchants have been asking us for a quick way to record a transaction in the app with minimal information – like with only a charge amount. Maybe you’re at a farmer’s market and simply want to charge a customer’s credit card and don’t need to manage inventory. This release includes a new “Simple Payments” feature on the orders tab. If you’re a WooCommerce Payments customer, you’ll find this works great with the In-Person Payments credit card reader! Don’t have a credit card reader yet? Alternatively, you easily send a payment link to your customer to pay with their device through your site.\n"
+"You’ll notice a new tab called “Menu” in this release. We’ve moved store settings, switching between different stores, and viewing WooCommerce Admin to that screen. More functionality will appear there over the next couple of months!\n"
```

Comment:
Almost got caught by those quotes, but those are curly quotes, so no need to escape them after all 👍

Reply:
Too late now, but isn't the use of contractions like `You'll` and `We've` something to avoid in editorialized copies which are supposed to use a bit more formal English? 🤷  Maybe next time 🙃